shell/browser/native_window_views_win.cc

```diff
@@ -301,6 +301,15 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
       }
       return false;
     }
+    case WM_NCACTIVATE: {
+      // Chromium renders titlebar as active as long as there is a views window
+      // active, which is different from most other win32 apps. We prevent this
+      // behavior by overriding the WM_NCACTIVATE message to default behavior.
+      // See also https://github.com/electron/electron/issues/24647.
+      *result =
```

Comment:
Can't we instead override [HWNDMessageHandlerDelegate::ShouldPaintAsActive()](https://source.chromium.org/chromium/chromium/src/+/master:ui/views/win/hwnd_message_handler.cc;l=2093) to `false` which would still resort to the default behavior because of `paint_as_active || active`?

Reply:
This is a good idea 👍 , I have update the PR.